.circleci/workflows.yml

```diff
@@ -1985,7 +1985,7 @@ jobs:
       <<: *defaultsParameters
       resource_class:
         type: string
-        default: large
+        default: xlarge
```

Comment:
We're doing a lot in the binary build now. `xlarge` is needed to not run out of memory.